### PR TITLE
Implement CSV ConfigStore in Node.js SDK

### LINKED ISSUE #151: Implement `CSV` `ConfigStore` in Node.js SDK
**Description**

- Plan a `Config` accommodation in the csv file.
- Add `CSVConfigStore` class to the Node.js SDK stores.
- A quick search indicates that [csv](https://www.npmjs.com/package/csv) library or [xlsx](https://www.npmjs.com/package/xlsx) are the most "suitable" for use during implementation.
- We would be happy to hear more suggestions, or we can implement the APIs ourselves.

**Motivation**

Extending Configu's [available stores](https://configu.com/docs/config-store/#available-stores) collection.
This ConfigStore will be made available in the Configu CLI since the Node.js SDK is used internally.

**Additional context**

- https://en.wikipedia.org/wiki/Comma-separated_values
- https://configu.com/docs/config-store/
- https://github.com/configu/configu/blob/main/types/ConfigStore.ts
- https://github.com/configu/configu/tree/main/ts/packages/node/src/stores

image

### PR DESCRIPTION
**Pull request by Maran1947**
Closes #151 
Hi @RichardAkman, Can you please take a moment to review it?

### CODE REVIEW

File: ts/packages/node/src/stores/CsvFile.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  parse(fileContent: string): Config[] {
+    const csvRows = parse(fileContent);
+    const configs = csvRows.slice(1).map(([set, key, value]: [string, string, string]) => ({ set, key, value }));
+    return configs;
+  }
+
+  stringify(nextConfigs: Config[]): string {
+    const csvRows = [['set', 'key', 'value'], ...nextConfigs.map(({ set, key, value }) => [set, key, value])];
+    return stringify(csvRows);
+  }
```
Comment (RichardAkman) [inline ts/packages/node/src/stores/CsvFile.ts:22]: We should utilize the capabilities of the `csv` package to streamline these methods:

```suggestion
  parse(fileContent: string): Config[] {
    return parse(fileContent, { columns: true });
  }

  stringify(nextConfigs: Config[]): string {
    return stringify(nextConfigs, { header: true });
  }
```

**Review by RichardAkman — CHANGES_REQUESTED** [review]
Great work, your experience implementing config stores is really showing ⭐ 

Take a look at my suggestion and then we can finalize this PR 🚀

**Comment by Maran1947** [comment]
Thanks for appreciation. I've really enjoyed working on config stores, and your support means a lot to me! 🙌 🙏

I've noticed that the `parse` method breaks when encountering empty lines, so I've added a `skip_empty_lines` attribute to handle this issue. Additionally, I've included a `trim` attribute to handle spaces and tabs. 

Could you please review these changes and suggest any improvements?

**Review by RichardAkman — APPROVED** [review]
I've removed the addition of `trim` as we do not want to mutate the data that the users of the config store are storing. Looks great now, the addition of `skip_empty_lines` is solid. 🏅

**Pull request merged**